src/Mono.Android/Xamarin.Android.Net/AndroidMessageHandler.cs

```diff
@@ -157,7 +157,7 @@ public CookieContainer CookieContainer
 
 		public bool CheckCertificateRevocationList { get; set; } = false;
 
-		public Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> ServerCertificateCustomValidationCallback { get; set; }
+		public Func<HttpRequestMessage, X509Certificate2?, X509Chain?, SslPolicyErrors, bool>? ServerCertificateCustomValidationCallback { get; set; }
```

Comment:
If users never set this, would the changes here be a no-op? There is a lot of certificate loading here, and I'm wondering how that will impact performance.

Reply:
This will be a no-op now.